pipes/WDL/tasks/tasks_assembly.wdl

```diff
@@ -195,6 +195,18 @@ task ivar_trim {
 
     String  bam_basename=basename(aligned_bam, ".bam")
 
+    meta {
+      description: "this runs ivar trim on aligned reads, which results in soft-clipping of alignments"
+    }
+
+    parameter_meta {
```

Comment:
I wonder if the `parameter_meta` values aren't being pulled in because they need to be in the more verbose form shown in the [`WDL-AID` readme](https://github.com/biowdl/wdl-aid/blob/master/docs/index.rst#basic-usage)
```
parameter_meta {
    input_name: {
        description: "A description of what value should be provided and is what it is used for.",
        category: "required"
    }
}
```
Glancing at the WDL-AID source, [this appears to be the case](https://github.com/biowdl/wdl-aid/blob/master/src/wdl_aid/wdl_aid.py#L181-L209); if no `description` key is present it seems to fall back to using the value passed in at call time from the `--fallback-description` arg.

Reply:
That more verbose form is WDL 1.0, so it won’t compile (Travis tests fail) on our draft-2 workflows. I figure I’d leave that until we sort out 1.0 more generally.